### PR TITLE
[lunar] Add realtime_tools, control_toolbox and control_msgs to doc&source jobs

### PR DESCRIPTION
**Pull request by bmagyar**
This fixes the jobs of ros_control(lers)

### CODE REVIEW

File: lunar/distribution.yaml [hunk 1]
```diff
@@ -176,6 +176,26 @@ repositories:
...
+  control_toolbox:
+    doc:
+      type: git
+      url: https://github.com/ros-controls/control_toolbox.git
+      version: kinetic-devel
+    source:
+      type: git
+      url: https://github.com/ros-controls/control_toolbox.git
+      version: kinetic-devel
+status: maintained
```
Comment (tfoote) [inline lunar/distribution.yaml]: This indentation is incorrect

**Comment by bmagyar** [comment]
Sorry! Should be ok now

File: lunar/distribution.yaml [hunk 2]
```diff
@@ -176,6 +176,26 @@ repositories:
...
+  control_toolbox:
+    doc:
+      type: git
+      url: https://github.com/ros-controls/control_toolbox.git
+      version: kinetic-devel
+    source:
+      type: git
+      url: https://github.com/ros-controls/control_toolbox.git
+      version: kinetic-devel
+    status: maintained
```
Comment (tfoote) [inline lunar/distribution.yaml]: Trailing whitespace here

File: lunar/distribution.yaml [hunk 3]
```diff
@@ -679,6 +699,16 @@ repositories:
       url: https://github.com/ros/rospack.git
       version: lunar-devel
     status: maintained
+  realtime_tools:
```
Comment (tfoote) [inline lunar/distribution.yaml]: This is out of alphabetical order

**Review by tfoote — CHANGES_REQUESTED** [review]
There's at least some trailing whitespace, as well as one section out of order. There's a diff in the CI block6 you can look at to fix it.

**Comment by bmagyar** [comment]
Ah this is getting embarrassing... I'll add 3 separate commits to see where it breaks.

**Comment by bmagyar** [comment]
Finally...

**Comment by vrabaud** [comment]
I was waiting in front of the merge button for hours to get it ready !

**Comment by bmagyar** [comment]
Much appreciated ;)

**Pull request merged**